screens/OrderItemDetailEditModal.js

```diff
@@ -328,19 +340,76 @@ class OrderItemOptions extends React.Component {
             this.setState({optionActiveSections: activeSectionsArr})
         }
+        if (this.props?.initialValues.sku) {
+            this.setState({selectedSku: this.props?.initialValues.sku})
+        }
+        if (this.props?.initialValues.quantity) {
+            this.setState({selectedSkuQuantity: this.props?.initialValues.quantity})
+        }
+    }
+
+    InventoryItem = (item) => {
+        //"baseUnitQuantity": 2,
+        // "minimumStockLevel": 9,
+        // "name": "F",
+        // "quantity": 88,
+        // "sku": "Test2",
+        // "unitOfMeasure": "R",
+        let isSelected = this.state.selectedSku == item.item.sku
+            || this.state.highlightSkuIndex == item.index
+
+
+        return (
+            <View style={[{flex: 1, marginHorizontal: 20}]}>
+                <TouchableHighlight
+                    style={{flexDirection: 'row', justifyContent: 'space-between', color: this.contetx?.customMainThemeColor}}
+                    onPress={() => {
+                        this.props.change(`sku`, item?.item.sku)
+                        this.setState({highlightSkuIndex: item.index, selectedSku: item?.item.sku})
+
+                        if (!!isSelected) {
+                            this.props.change(`quantity`, this.state.selectedSkuQuantity + 1)
+                            this.setState({selectedSkuQuantity: (this.state.selectedSkuQuantity + 1)})
+                        }
+                        console.log(this.state.selectedSku, this.props.sku)
+                    }}
+                >
+                    <View style={[styles.flexButton(isSelected ? this.context?.customMainThemeColor : '#fff8c9')]}>
+
+                        <View style={{color: isSelected ? '#fff' : '#555'}}>
+                            <StyledText style={{color: isSelected ? '#fff' : '#555', paddingTop: 8, paddingBottom: 4, fontWeight: 'bold', fontSize: 16}}>{item?.item.sku}</StyledText>
+                        </View>
+                        <View>
+                            <StyledText style={{color: isSelected ? '#fff' : '#555', paddingBottom: 8}}>{item?.item.name}</StyledText>
+                        </View>
+                        <View style={{borderTopWidth: 1, borderColor: isSelected ? '#fff' : '#ccc', backgroundColor: isSelected ? this.context?.customMainThemeColor : '#fff8c9', width: '100%', justifyContent: 'center', alignItems: 'center', paddingVertical: 4}}>
+                            <StyledText style={{color: isSelected ? '#fff' : '#555'}}>{item?.item.quantity}</StyledText>
+                        </View>
+                    </View>
+
+                </TouchableHighlight>
+            </View >
+        );
     }
 
     render() {
         const {product, globalProductOffers} = this.props
         const {t, themeStyle, customMainThemeColor} = this.context
+        // console.log("order modal: ", Object.values(product.inventory.inventoryQuantities))
```

Comment:
Please remember to remove all commented out code by default unless there is specific reasons to keep them, such as for documentation purpose